src/main/java/com/google/jenkins/plugins/k8sengine/KubeConfig.java

```diff
@@ -91,7 +97,7 @@ private void setCurrentContext(String currentContext) {
   public String toYaml() throws IOException {
     return new Yaml()
         .dumpAsMap(
-            new ImmutableMap.Builder<String, Object>()
+            new MapBuilder()
```

Comment:
I'm surprise to see the compiler complaining about the `put` method.

Reply:
compiler warnings fixed